docs/user-guides/desktop/index.md

```diff
@@ -0,0 +1,82 @@
+# Coder Desktop
+
+Use Coder Desktop to work on your workspaces as though they're on your LAN, no
+port-forwarding required.
+
+## Install Coder Desktop
+
+<div class="tabs">
+
+You can install Coder Desktop on macOS or Windows.
+
+### macOS
+
+1. Use [Homebrew](https://brew.sh/) to install Coder Desktop:
+
+   ```shell
+   brew install --cask coder/coder/coder-desktop
+   ```
+
+1. Open Coder Desktop from the Applications directory and when macOS asks if you want to open it, select **Open**.
+
+1. The application connects as a VPN. macOS will prompt you to confirm with:
+
+   **"Coder Desktop" would like to use a new network extension**
+
+   Select **OK**.
+
+1. The Coder icon will appear in the macOS menu bar:
+
+   ![Coder icon and menu](../../images/user-guides/desktop/coder-desktop-mac-menu-bar.png)
+
+1. Continue to the [configuration section](#configure).
+
+### Windows
+
+Download from there.
+
+</div>
+
+## Configure
+
+Before you can use Coder Desktop, you will need to approve the system extension and log in.
```

Comment:
Do we want to move this to a macOS-specific heading or tab?
I'm guessing that the Windows configuration steps will be different.

Reply:
I adjusted the step per @ethanndickson 's suggestion and that moved the configuration step into the mac-specific section

I think the rest of the configuration _should_ be the same, but I haven't been able to try the Windows version yet